### PR TITLE
chore: update builder to support quick build

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Part of https://github.com/GoogleChromeLabs/chromium-bidi/issues/2240

### CODE REVIEW

File: tools/wpt-report-builder/builder.mjs [hunk 1]
```diff
@@ -55,20 +84,41 @@ function getChromeVersion() {
...
+ * @param {string[]} filePaths
+ */
+function readReport(filePaths) {
+  const results = [];
+  for (const filePath of filePaths) {
+    if (!fs.existsSync(filePath)) {
+      continue;
+    }
+    const json = fs.readFileSync(filePath, 'utf8');
+    // File may be empty if the interop test shard failed
```
Comment (Lightning00Blade) [inline tools/wpt-report-builder/builder.mjs:98]: Needed as due to the large number of shards, WPT fails to find tests.

File: .github/workflows/wpt-quick.yml [hunk 2]
```diff
@@ -113,7 +113,7 @@ jobs:
         if: ${{matrix.kind == 'chromedriver'}}
         run: >
           xvfb-run --auto-servernum
-          npm run wpt -- "webdriver/tests/bidi/${{ github.event.inputs.tests }}"
+          npm run wpt -- "webdriver/tests/interop/${{ github.event.inputs.tests }}" || true
```
Comment (Lightning00Blade) [inline .github/workflows/wpt-quick.yml:116]: I need to see what do about this one as we want to fail if a test fails.

File: .github/workflows/wpt-quick.yml [hunk 3]
```diff
@@ -113,7 +113,7 @@ jobs:
         if: ${{matrix.kind == 'chromedriver'}}
         run: >
           xvfb-run --auto-servernum
-          npm run wpt -- "webdriver/tests/bidi/${{ github.event.inputs.tests }}"
+          npm run wpt -- "webdriver/tests/interop/${{ github.event.inputs.tests }}"
```
Comment (Lightning00Blade) [inline .github/workflows/wpt-quick.yml:116]: We missed this part, so we were running the test 2 times

**Comment by Lightning00Blade** [comment]
The removal of the full runner will be done in a separate PR.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**